charts/microgateway/Chart.yaml

```diff
@@ -21,17 +21,19 @@ maintainers:
   - email: support@airlock.com
     name: Airlock
 name: microgateway
-version: 3.0.14
+version: 3.0.15
```

Comment:
The release Helm Chart 3.0.14 is still in progress, we will published it with Airlock Microgateway 3.2.  Therefore we can not increment the version number.

Reply:
done